code/__DEFINES/role_preferences.dm

```diff
@@ -101,14 +101,14 @@
 #define ROLE_HERMIT "Hermit"
 #define ROLE_HOTEL_STAFF "Hotel Staff"
 #define ROLE_LAVALAND_SYNDICATE "Lavaland Syndicate"
+#define ROLE_SPACE_SYNDICATE "Space Syndicate" //probably a ghost role
```

Comment:
unmarked edit, also not needed to just undo it

Reply:
done i think